contrib/folly_memcpy/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+if(UNIX AND NOT APPLE)
```

Comment:
a nit: it is interesting we use `APPLE` as operating system, for instance here we compare it with `UNIX`; alternate would have been `UNIX ANS NOT MAC`. 
No action needed in this PR, just sharing a thought.